.github/workflows/CI.yml

```diff
@@ -123,13 +123,21 @@ jobs:
       - name: "glib: build nightly"
         run: cargo build --manifest-path glib/Cargo.toml --all-features
         if: matrix.rust == 'nightly'
+      - name: "glib: tests glib macros nightly"
+        working-directory: glib
+        run: python3 clone_tests/clone_compilation_errors.py
+        if: matrix.rust == 'nightly'
```

Comment:
Why only on nightly?

Reply:
Arf! I forgot to remove this check on the second.